types/src/front/lib/assistant.ts

```diff
@@ -331,7 +331,10 @@ const ANTHROPIC_TOOL_USE_META_PROMPT =
   `After using a tool, think for one short bullet point in \`<thinking>\` tags to evaluate ` +
   `whether the tools results are enough to answer the user's question. ` +
   `The response to the user must be in \`<response>\` tags. ` +
-  `There must be a single \`<response>\` after the tools use (if any).`;
+  `There must be a single \`<response>\` after the tools use (if any). ` +
```

Comment:
Could we instead have this in `@claude-3` assistant prompt ? This clutters meta prompt even more and references a tool that may not even exist for the agent @spolu @overmode

Reply:
Agreed. Let's make it something specific to the global agent.

Reply:
I just tried to create a super simple personal assistant with browsing capabilities running on Claude and without the updated meta prompt. Asking for the capital of Paris triggers a web search.
-> Seems like all claude models with search will need this guidance:  meta prompt seems to make sense, or I could add this line in the search function description. I think tool makes more sense to leave the tool description behavior-agnostic.

Reply:
You definitely don't want this meta-prompt if the assistant doesn't have any search tool

Reply:
If you create your own assistant and give it websearch, and you realize it searches the web more than you wish, you can always edit the instructions

Reply:
I see, what you're suggesting, editing the instructions, will be needed every time you create a claude assistant with browsing capabilities, which is quite annoying and will probably look clunky to the user. But still I'm fine with keeping this meta prompt untouched because it does feel weird to give instructions about unknown tools. 
Will be thinking about a way to correct the behavior, let me know if you think of anything.

Reply:
We could potentially have a "websearch_metaprompt" that gets added only if you have websearch.

That being said, if you want to make a perplexity-like assistant, you don't want this metaprompt at all, you just want the assisstant to always search even for general knowledge (better answers, reduce hallucinations etc...).
I don't think we should be making this choice on behalf of users. Otherwise the users might start adding "always search the web for every question" while we add "don't search the web for general knowledge", which is likely to impact performance negatively

Reply:
That sounds right, an explanation I'd accept as a user I think, thanks!

Reply:
Anyway, my stance is that hidden meta prompts are evil and trapdoory (they are hard to remove and change, you don't know the impact adding/removing/changing them has on assistants, users don't know about them etc..)

we should only add them when really necessary